agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.shardingsphere.agent.plugin.core.util;
```

Comment:
Missing License.

Reply:
Thanks, I've added the license in this file.